php-resource/project.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name" : "sample-php-resource",
+    "id" : "sample-php-resource",
+    "projectType" : "custom",
+    "sourceLocale" : "en-US",
+    "pseudoLocale" : "zxx-XX",
+    "resourceDirs" : {
+        "javascript" : "resources"
+    },
+    "resourceFileTypes" : {
+        "javascript" : "ilib-loctool-php-resource"
+    },
+    "plugins" : [
+        "ilib-loctool-javascript"
+    ],
+    "excludes" : [
+        ".*",
+        "xliffs",
+        "resources"
+    ],
+    "settings" : {
+        "xliffsDir" : "./xliffs",
+        "locales" : [
+            "ko-KR",
+            "de-DE"
+        ],
+        "javascript" : {
+            "wrapper" : "\\.\\s*getString(JS)?",
+            "mappings": {
+                "**/*.js": {
+                    "template": "resources/Strings[locale].php"
+                }
+            }
+        },
```

Comment:
Gotta say I don't exactly understand why do we map from `.js` to `.php` and overall how are we supposed to ouptut php files using this config, any chance you could a readme describing that?

Reply:
And maybe also commit the produced php files as well

Reply:
We should not commit the resulting php files, especially when this becomes an E2E test. We should commit  snapshots though.

Added a short description in the README.